components/reddit/reddit.app.js

```diff
@@ -246,9 +249,9 @@ module.exports = {
             title,
             name,
           });
-        });
+        }); 
       } while (after);
       return results;
     },
   },
-};
+};
```

Comment:
Similar to the trailing whitespaces setttings, you should add the following setting so that a newline is added at the end of the file:

```json
{
  "files.insertFinalNewline": true
}
```

Reply:
Ah, ok, added this setting to my VSCode settings. Should be good.